samples/saga/ravendb-sequence-generator/Version_5/Sample/Program.cs

```diff
@@ -0,0 +1,49 @@
+﻿using NServiceBus;
+using NServiceBus.Persistence;
+using Raven.Client;
+using Raven.Client.Document;
+using System;
+using System.Threading.Tasks;
+
+class Program
+{
+
+    static void Main()
+    {
+        BusConfiguration busConfig = new BusConfiguration();
+		busConfig.EndpointName( "Samples.RavendbSequenceGenerator" );
+        busConfig.UseSerialization<JsonSerializer>();
+        busConfig.EnableInstallers();
```

Comment:
@mauroservienti we could enforce here the idea of conditional execution to make sure people don't do this in production all the time. Example [here](https://github.com/Particular/NServiceBus.Msmq.Samples/blob/master/VideoStore.Msmq/VideoStore.ECommerce/Global.asax.cs#L33-39)